test/invalidateall.test.js

```diff
@@ -0,0 +1,80 @@
+const request = require('supertest'),
+  app = require('../app.js'),
+  { factory } = require('factory-girl'),
+  { hashPassword } = require('../app/helpers/hasher'),
+  correctPassword = 'password',
+  correctEmail = 'invalidateall@wolox.com.ar',
+  correctEmail2 = 'invalidateall2@wolox.com.ar',
+  permissionErrorStatus = 401,
+  User = require('../app/models').user;
+
+describe('POST /users/sessions/invalidate_all', () => {
+  test('Test create two users, log in, invalidate al sessions from one of them', () => {
+    let token = '';
+    return hashPassword(correctPassword)
+      .then(pass =>
+        factory
+          .create('user', {
+            email: correctEmail,
+            password: pass
+          })
+          .then(() =>
+            factory.create('user', {
```

Comment:
You could have used `Promise.all()` here. Also, even if there are no other test to reuse it, its often better to move the test setup to a `beforeEach()`, to help make long tests more readable by separating the running of the endpoint itself and the setup